Druid/DRJITMessageSend.class.st

```diff
@@ -42,7 +42,7 @@ DRJITMessageSend >> isSpecialSelectorSend [
 { #category : #accessing }
 DRJITMessageSend >> numberOfArguments [
 	
-	^ operands second value
+	^ operands second compiledValue 
```

Comment:
This kind of extends from before, right?
We can either have here constants or jit compile time expressions, right?